src/include/btree.i

```diff
@@ -945,12 +915,14 @@ __wt_page_release_busy(WT_SESSION_IMPL *session, WT_REF *ref, uint32_t flags)
 	 */
 	locked = WT_ATOMIC_CAS4(ref->state, WT_REF_MEM, WT_REF_LOCKED);
 	WT_TRET(__wt_hazard_clear(session, page));
-	if (!locked)
+	if (!locked) {
+		WT_TRET(EBUSY);
```

Comment:
This is the only part of the change that I'm suspicious about. Could we be returning an EBUSY to the direct caller of ```__wt_page_release_evict``` when we returned success before?